packages/code-block/partials/code-tabs/index.test.js

```diff
@@ -1,14 +1,15 @@
 import { fireEvent, render, screen, waitFor } from '@testing-library/react'
 import CodeTabs from './'
 import CodeBlock from '../../'
+import CodeBlockConfig from '../code-block-config'
 import CodeTabsProvider from '../../provider'
 
 // getAllByText is needed, as we sometimes render dummy elements
 // before the main, displayed element, eg for overflow detection
 // and for simpler copy-to-clipboard logic
 function getSecondByText(text) {
   /* eslint-disable-next-line no-unused-vars */
-  const [firstElem, secondElem] = screen.getAllByText(text)
+  const [_firstElem, secondElem] = screen.getAllByText(text)
```

Comment:
Nit, this is an unused variable, prefixing with `_` makes VSCode happy (at least in my case).

Reply:
If you want I think you could update this to not name any variable at all:

```suggestion
  const [, secondElem] = screen.getAllByText(text)
```

Reply:
nice, TIL!